<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockDialog</class>
 <widget class="QDialog" name="ClockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 71, 71);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDateTimeEdit" name="clockTimeedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vivaldi</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>12</month>
     <day>22</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d H:mm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>176</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="timeCkeck_1">
      <property name="font">
       <font>
        <family>Vivaldi</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000/00/00 00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="timeCkeck_2">
      <property name="font">
       <font>
        <family>Vivaldi</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000/00/00 00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="timeCkeck_3">
      <property name="font">
       <font>
        <family>Vivaldi</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000/00/00 00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="timeCkeck_4">
      <property name="font">
       <font>
        <family>Vivaldi</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0000/00/00 00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>111</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="todayButton">
      <property name="text">
       <string>日历</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
